src/annotator/anchoring/pdf.js

```diff
@@ -249,10 +275,11 @@ function findPage(offset) {
  * scrolled into view.
  *
  * @param {number} pageIndex - The PDF page index
- * @param {TextPositionAnchor} anchor - Anchor to locate in page
+ * @param {number} start - Character offset within the page's text
+ * @param {number} end - Character offset within the page's text
  * @return {Promise<Range>}
  */
-async function anchorByPosition(pageIndex, anchor) {
+async function anchorByPosition(pageIndex, start, end) {
```

Comment:
This removes the `TextPositionAnchor` dependency since only the `start` and `end` fields were needed.